string topic_name
---
bool extracted
string data_json

